managed/CounterStrikeSharp.API/Modules/Admin/BaseRequiresPermissions.cs

```diff
@@ -27,13 +27,15 @@ public BaseRequiresPermissions(params string[] permissions)
             Command = "";
         }
 
-        public virtual bool CanExecuteCommand(CCSPlayerController? caller)
+        public virtual bool CanExecuteCommand(SteamID? steamID)
```

Comment:
Is this a public API? We might need to keep the old prototype around for backwards compatibility, and then proxy it to the new method using `caller.AuthorizedSteamID`